lib/features/children/details/pages/child_details_page.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/children/details/cubit/child_details_cubit.dart';
+import 'package:givt_app/features/children/details/widgets/child_details_item.dart';
+import 'package:givt_app/features/children/details/widgets/child_giving_allowance_card.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+
+class ChildDetailsPage extends StatelessWidget {
+  const ChildDetailsPage({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final locals = AppLocalizations.of(context);
+    return BlocBuilder<ChildDetailsCubit, ChildDetailsState>(
```

Comment:
I would try to not wrap the whole page into a builder but try to go as deep as possible

Reply:
goop point in terms of optimisation, however I need builder on that level since I use state for scaffold's app bar.